### PR TITLE
replace jumpto.js with skipto.js version 4.2 

### PR DESCRIPTION
**Pull request by jongund**
This pull request changes the code base from using JumpTo.js to SkipTo.js.

The 4.2 version of SkipTo.js has been updated to fix the following issues:
1. Removing copyright code concerns by using the skipto.js script instead of the jumpto.js script,  see [SkipTo copyright statement](https://raw.githubusercontent.com/w3c/wai-aria-practices/dev/skipto/ARIA/apg/example-index/js/skipto.js) in code.
2. Selection of main landmark option on the home page, by insuring focus moves to a visible element.
3. The Skip To menu button labeling issue (e..g. "opton" is now pronounced instead of "command" on macOS).
4. The skipto menu of options will now scroll if the number of items exceeds the size of the window to display them all.

### CODE REVIEW

**Review by howard-e — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (howard-e) [inline package-lock.json:1]: @jongund I think pushing this `package-lock.json` may have been a mistake?

**Comment by mcking65** [comment]
@richnoah  Idon't concur with latest changes by Jon; I need to fix the license/copyright to be what is appropriate based on agreement with skipto.

File: package-lock.json (hunk 1 above)
Comment (jongund) [reply package-lock.json:1]: Sorry, should not be part of pull request, my applogies

Comment (howard-e) [reply package-lock.json]: @jongund No problem. I've gone ahead and removed it.

**Review by mcking65 — APPROVED** [review]
Thank you all; this is good to ship now. I will merge.

**Pull request merged**